sdk/include/opentelemetry/sdk/trace/async_batch_span_processor.h

```diff
@@ -25,20 +25,7 @@ namespace trace
  */
 struct AsyncBatchSpanProcessorOptions
 {
-  /**
-   * The maximum buffer/queue size. After the size is reached, spans are
-   * dropped.
-   */
-  size_t max_queue_size = 2048;
-
-  /* The time interval between two consecutive exports. */
-  std::chrono::milliseconds schedule_delay_millis = std::chrono::milliseconds(5000);
-
-  /**
-   * The maximum batch size of every export. It must be smaller or
-   * equal to max_queue_size.
-   */
-  size_t max_export_batch_size = 512;
+  BatchSpanProcessorOptions options;
```

Comment:
Just want to discuss it, is it simpiler to declare `struct AsyncBatchSpanProcessorOptions : public BatchSpanProcessorOptions` ?

Reply:
Thanks, this is much better.